### PR TITLE
Fix export genesis for historical height

### PR DESCRIPTION
**Pull request by yzang2019**
## Describe your changes and provide context
Add a new function for baseapp LoadVersionWithoutInit so that we allow genesis export to export historical height after app.New is called.

### CODE REVIEW

File: storev2/commitment/store.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ func (st *Store) Set(key, value []byte) {
 
 // Implements types.KVStore.
 func (st *Store) Get(key []byte) []byte {
+	fmt.Printf("Getting key %s, current version is %d\n", string(key), st.tree.Version())
```
Comment (udpatil) [inline storev2/commitment/store.go:97]: we probably want to remove prints, right?

File: storev2/rootmulti/store.go [hunk 2]
```diff
@@ -364,14 +366,9 @@ func (rs *Store) LoadVersionAndUpgrade(version int64, upgrades *types.StoreUpgra
...
 	}
-	if err := rs.scStore.Initialize(initialStores); err != nil {
-		return err
-	}
-	if version > 0 {
-		_, err := rs.scStore.LoadVersion(version, false)
-		if err != nil {
-			return nil
-		}
+	rs.scStore.Initialize(initialStores)
```
Comment (udpatil) [inline storev2/rootmulti/store.go:369]: this can still error, right? if so, don't we still need to return?

**Review by udpatil — APPROVED** [review]

File: storev2/rootmulti/store.go (hunk 2 above)
Comment (yzang2019) [reply storev2/rootmulti/store.go:369]: This will not error after the seidb change

Comment (yzang2019) [reply storev2/rootmulti/store.go:367]: https://github.com/sei-protocol/sei-db/pull/64/files

**Pull request merged**